README.md

```diff
@@ -41,6 +41,7 @@ _(starting Sidekiq `v3.3.1`)_
 | sidekiq_queue_workers                     | gauge   | The number of workers serving the queue (labels: `name`)                                               |
 | sidekiq_queue_processes                   | gauge   | The number of processes serving the queue (labels: `name`)                                             |
 | sidekiq_queue_busy_workers                | gauge   | The number of workers performing the job for the queue (labels: `name`)                                |
+| sidekiq_host_processes                    | gauge   | The number of quiet/active processes running on the host (labels: `host`, `quiet`)                     |
```

Comment:
This put us in a position that we will have to raise the bar for the Sidekiq version, because the version a quiet attribute appears is [4.1.0](https://github.com/sidekiq/sidekiq/blob/v4.1.0/lib/sidekiq/api.rb#L700). Because of that we will have to drop support of the versions 3.x, but I think that's fine.

Also I would like to ask you to move it right after the `sidekiq_processes` gauge and remove from description mention of `quiet/active` because it's just labels and we can have more than that for this metric in the future.

Reply:
Thanks for your prompt answer!

Fixed!